data/config-schema/src/main/resources/net/corda/schema/configuration/cryptoLibrary/1.0/corda.cryptoLibrary.json

```diff
@@ -5,6 +5,41 @@
   "description": "Configuration schema for the crypto library subsection.",
   "type": "object",
   "properties": {
+    "unmanagedWrappingKeys": {
+      "description" : "Key derivation parameters for unmanaged wrapping keys",
+      "type": "object",
+      "default": {},
+      "patternProperties": {
+        ".*": { 
+          "type" :  "object",
+          "properties": {
```

Comment:
Do we need a type (with the default our only supported type `AES-...`)?
Maybe that's premature, on the other hand, maybe it helps being explicit? We can even have it as the only allowed value in the schema?

Reply:
Probably a symbolic string for key derivation function and key type. Sounds good.